src/ActiveLogin.Identity.Swedish.TestData/TestData.fs

```diff
@@ -0,0 +1,49 @@
+namespace ActiveLogin.Identity.Swedish.FSharp.TestData
+open ActiveLogin.Identity.Swedish.FSharp
+open ActiveLogin.Identity.Swedish.TestData.AllPins
+open System
+
+
+module SwedishPersonalIdentityNumberTestData =
+    let private rng = Random()
+    let private random _ = rng.Next()
+    let internal shuffledPins() = allPins |> Array.sortBy random
```

Comment:
Can this be made a seq (which I guess is equvivalent of IEnumerable)? Might improve perf when only accessing parts of the data.

Reply:
I thought sortBy on an IEnumerable would require a full enumeration to sort the collection, right? And in that case it is more honest to return an array than an IEnumerable, because the client will have to take the full performance hit of shuffle even if they just want a small part of the data. 

But I will benchmark it. 

And if you can think of another algorithm to shuffle an array I would be happy to change it.

Reply:
|               Method |      N |     Mean |    Error |    StdDev |
|--------------------- |------- |---------:|---------:|----------:|
|    Get4000RandomPins | 100000 | 375.0 ms | 6.897 ms |  6.452 ms |
| Get4000RandomPinsSeq | 100000 | 386.7 ms | 7.659 ms | 12.796 ms |

No performance gain by just making it IEnumerable (since sortBy still enumerates the full collection). But maybe there is some perf gain to be had with another shuffle algorithm.